frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.html

```diff
@@ -3,122 +3,130 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 <div class="dialog">
-  <div class="header">
+  <div class="header relative">
     <h2>View diagrams</h2>
-    <span
-      >Last update:
+    <div class="mb-2">
+      Last update:
       {{
         (diagramMetadata?.last_updated | date: "EE, dd MMM y HH:mm:ss") ||
           "loading..."
-      }}</span
-    >
+      }}
+    </div>
 
-    <mat-form-field
-      style="margin-bottom: -1.25em"
-      id="search"
-      appearance="outline"
-    >
-      <mat-label>Search</mat-label>
-      <input
-        [(ngModel)]="search"
-        autocomplete="off"
-        matInput
-        placeholder="Diagram name or uuid"
-      />
-      <mat-icon matSuffix>search</mat-icon>
-    </mat-form-field>
-    <span *ngIf="filteredDiagrams()"
-      >{{ filteredDiagrams()?.length }} diagram(s) found:
-    </span>
-  </div>
-  <div class="flex flex-wrap">
-    <mat-card
-      class="diagram-card"
-      #diagram
-      [id]="diagram.uuid"
-      [attr.success]="diagram.success"
-      *ngFor="let diagram of filteredDiagrams()"
-    >
-      <img
-        (click)="openModelDiagramPreviewDialog(diagram)"
-        matTooltip="Open preview"
-        *ngIf="diagram.success && diagrams[diagram.uuid]?.content"
-        class="diagram"
-        [src]="diagrams[diagram.uuid].content"
-      />
+    <app-model-diagram-code-block
+      [project]="data.project"
+      [model]="data.model"
+    ></app-model-diagram-code-block>
+
+    <div>
```

Comment:
Please check whether this empty div here makes sense

Reply:
Is removed.